packages/lab/src/parent-child-layout/ParentChildLayout.tsx

```diff
@@ -1,39 +1,33 @@
-import {
-  Breakpoints,
-  FlexLayout,
-  FlexItem,
-  FlexLayoutProps,
-  makePrefixer,
-} from "@salt-ds/core";
-
+import { Breakpoints, makePrefixer } from "@salt-ds/core";
 import { clsx } from "clsx";
-import { forwardRef, ReactNode, useEffect } from "react";
+import {
+  ComponentPropsWithoutRef,
+  forwardRef,
+  ReactNode,
+  useEffect,
+} from "react";
 import { useIsViewportLargerThanBreakpoint } from "../utils";
-
 import { useWindow } from "@salt-ds/window";
 import { useComponentCssInjection } from "@salt-ds/styles";
 
 import parentChildLayoutCss from "./ParentChildLayout.css";
 
 export type StackedViewElement = "parent" | "child";
```

Comment:
you can probably remove the type if it is not going to be used

Reply:
removed - thanks